test/Rings/orderings.jl

```diff
@@ -189,7 +189,8 @@ end
 end
  
 @testset "Polynomial Orderings comparison" begin
-   R, (x, y, z) = @inferred polynomial_ring(QQ, ["x", "y", "z"])
+   #R, (x, y, z) = @inferred polynomial_ring(QQ, ["x", "y", "z"])
+   R, (x, y, z) = polynomial_ring(QQ, ["x", "y", "z"])
```

Comment:
This can be reverted, I think?
```suggestion
   R, (x, y, z) = @inferred polynomial_ring(QQ, ["x", "y", "z"])
```